extension/content_dashboard.js

```diff
@@ -7,16 +7,73 @@ $(function() {
     airline = AES.getAirlineCode();
     server = AES.getServerName();
     chrome.storage.local.get(['settings'], function(result) {
-        settings = result.settings;
+      settings = result.settings;
 
-        displayDashboard();
+      if ((settings['flightInformationsFixed'] === undefined) || (settings['flightInformationsFixed'] !== 1)) {
+        fixAndDeleteInvalidData();
+      }
+
+      displayDashboard();
+      dashboardHandle();
+      
+      $("#aes-select-dashboard-main").change(function() {
         dashboardHandle();
-        $("#aes-select-dashboard-main").change(function() {
-            dashboardHandle();
-        });
+      });
     });
 });
 
+function fixAndDeleteInvalidData() {
+  chrome.storage.local.get(null, function (entries) {
```

Comment:
It might be nice to get the data into a `const` with an `await`. Fewer nesting levels make the code easier to read.

Reply:
can of course be written differently and also moved somewhere else if you want, in content_dashboard.js is already very large anyway

Reply:
Yeah you are right, it was just good good enough for temporary use. Do you want to change it?

Reply:
I would like you to change it, yes.